### PR TITLE
Fixes clown and mime pin mob sprites

### PR DESCRIPTION
**Pull request by cacogen**
## About the pull request

Gives them four directions, puts them in the right place on the directions they're visible. Previously they only had the one which was then used for all four directions.

__Also does the following:__
- Adds \improper to pin names
- Removes lines defining the same as the parent
- Adds punctuation to end of descriptions

## Why it's good for the game

Looks better

## Changelog
:cl: cacogen
fix: Clown and mime pin mob sprites now appear correctly from all angles
/:cl:

### CODE REVIEW

File: code/modules/clothing/under/accessories.dm [hunk 1]
```diff
@@ -348,12 +345,9 @@
 		SEND_SIGNAL(L, COMSIG_CLEAR_MOOD_EVENT, "fan_clown_pin")
 
 /obj/item/clothing/accessory/fan_mime_pin
-	name = "Mime Pin"
-	desc = "A pin to show off your appreciation for mimes and miming"
+	name = "\improper Mime Pin"
+	desc = "A pin to show off your appreciation for mimes and miming."
```
Comment (RaveRadbury) [inline code/modules/clothing/under/accessories.dm:349]:
```suggestion
	desc = "A pin to show off your appreciation for mimes and miming!"
```

Comment (RaveRadbury) [reply code/modules/clothing/under/accessories.dm:349]: Your bias is showing 😝

**Comment by RaveRadbury** [comment]
Thank you for taking the time to improve this.

File: code/modules/clothing/under/accessories.dm (hunk 1 above)
Comment (Fikou) [reply code/modules/clothing/under/accessories.dm:349]: mimes would not use something so emotional like !

**Review by Twaticus — APPROVED** [review]

**Pull request merged**